src/phoenix/server/main.py

```diff
@@ -46,10 +48,22 @@ def _get_pid_file() -> Path:
     return get_pids_path() / str(os.getpid())
 
 
+def _load_spans(
+    traces: Traces,
+    spans: Iterable[Span],
+    simulate_streaming: Optional[bool] = False,
+) -> None:
+    for span in spans:
+        if simulate_streaming:
+            sleep(random())
+        traces.put(span)
+
+
 if __name__ == "__main__":
     primary_dataset_name: str
     reference_dataset_name: Optional[str]
     trace_dataset_name: Optional[str] = None
+    simulate_streaming: Optional[bool] = None
```

Comment:
```suggestion
    simulate_streaming: Optional[bool] = False
```
Can't we also just have this as the fixture parser default?

Reply:
I was only following the convention from the line prior. Ether way it makes no difference, and we don't need to make an exception just for 'bool'.